website/docs/guides/migration/versions/00-upgrading-to-v1.7.md

```diff
@@ -28,7 +28,7 @@ This is a relatively small behavior change, but worth calling out in case you no
 
 ## New and changed features and functionality
 
-- [`dbt docs generate`](/reference/commands/cmd-docs) now supports `--select` to generate documentation for a subset of your project. Currently available for Snowflake and Postgres only, but other adapters are coming soon. 
+- [`dbt docs generate`](/reference/commands/cmd-docs) now supports `--select` to generate catalog metadata for a subset of your project. Currently available for Snowflake and Postgres only, but other adapters are coming soon. 
```

Comment:
What do you think about linking to the [Catalog JSON file](https://docs.getdbt.com/reference/artifacts/catalog-json) artifact page like this?
```suggestion
- [`dbt docs generate`](/reference/commands/cmd-docs) now supports `--select` to generate [catalog metadata](/reference/artifacts/catalog-json) for a subset of your project. Currently available for Snowflake and Postgres only, but other adapters are coming soon. 
```